data/json/items/magazine/weldgas.json

```diff
@@ -16,7 +16,8 @@
     "capacity": 60,
     "count": 60,
     "reliability": 10,
-    "flags": [ "NO_UNLOAD", "NO_RELOAD" ]
+    "flags": [ "NO_UNLOAD", "NO_RELOAD" ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "weldgas": 60 } } ]
```

Comment:
i don't know if weldgas is a gas or a liquid, but the pocket needs to be watertight or gastight depending.